### PR TITLE
A number of touch ups

### PR DESCRIPTION
**Pull request by captaincrazybro**

### CODE REVIEW

File: src/components/Datatable/DataRow.svelte [hunk 1]
```diff
@@ -1,3 +1,23 @@
...
+
+{#if redirect}
+  <style>
+    tr {
+      cursor: pointer;
+    }
+  </style>
+{/if}
+
+<tr on:click={handleOnClick} class="mdc-data-table__row">
```
Comment (forevermatt) [inline src/components/Datatable/DataRow.svelte:21]: You can simplify this to...
```suggestion
<tr on:click class="mdc-data-table__row">
```
... if you simply want to pass up the click even to the parent component.

**Review by forevermatt — APPROVED** [review]
Approved, with one suggestion

File: src/pages/home.svelte [hunk 2]
```diff
@@ -78,22 +82,29 @@ const menuItems = id => [
...
 let loading = false
+let otherClick = false
 let shownMenus = {}
 let gridCols = ''
 
 onMount(() => {
   setCardCols()
 })
 
+const redirect = (url) => {
```
Comment (hobbitronics) [inline src/pages/home.svelte:93]: () are not needed here just in case you didn't know.

File: src/pages/home.svelte [hunk 3]
```diff
@@ -78,22 +82,29 @@ const menuItems = id => [
 
 let selected = []
 let loading = false
+let otherClick = false
```
Comment (hobbitronics) [inline src/pages/home.svelte:85]: itemMenuClick or something would make the code below a little more readable

File: src/pages/items/[itemId]/new-claim.svelte [hunk 4]
```diff
@@ -4,7 +4,11 @@ import { claims, initialized, createClaim } from '../../../data/claims.js'
 import { goto, params } from '@roxi/routify'
 import { Button, Form, Page, TextArea } from '@silintl/ui-components'
```
Comment (hobbitronics) [inline src/pages/items/[itemId]/new-claim.svelte:5]:
```suggestion
import { Button, Form, Page, TextArea, Item } from '@silintl/ui-components'
```

File: src/pages/items/[itemId]/new-claim.svelte [hunk 5]
```diff
@@ -4,7 +4,11 @@ import { claims, initialized, createClaim } from '../../../data/claims.js'
...
 import { fade } from 'svelte/transition'
+import Item from '@silintl/ui-components/components/mdc/List/Item.svelte'
```
Comment (hobbitronics) [inline src/pages/items/[itemId]/new-claim.svelte:7]:
```suggestion
```

**Review by hobbitronics — APPROVED** [review]

**Pull request merged**